tests/caasp/stack_controller.pm

```diff
@@ -142,6 +142,17 @@ sub run {
     send_key "alt-tab";
     assert_screen 'xterm';
     type_string "export KUBECONFIG=~/Downloads/kubeconfig\n";
+
+    if (check_var('DISTRI', 'caasp') && get_var('VERSION', '2.0')) {
```

Comment:
I think you meant `check_var('VERSION', '2.0')` instead of `get_var('VERSION', '2.0')`. My `CaaSP 1.0` tests are executing this code, where I guess they shouldn't. In this regard, I am including a fix for this in PR#3609

Feel free to review it ;)